sufia-models/app/models/sufia/works/generic_work.rb

```diff
@@ -3,5 +3,6 @@ module Sufia::Works
   class GenericWork < Work
     include Sufia::Works::CurationConcern::WithBasicMetadata
     include Sufia::GenericFile::Permissions
+    include Sufia::GenericFile::Metadata
```

Comment:
I think we need only this line or 4, but not both.  I would suggest removing one and see if the tests still pass.